ios/ImagePickerManager.h

```diff
@@ -1,12 +1,11 @@
 #import <React/RCTBridgeModule.h>
 #import <UIKit/UIKit.h>
-#import <PhotosUI/PhotosUI.h>
 
 typedef NS_ENUM(NSInteger, RNImagePickerTarget) {
   camera = 1,
   library
 };
 
-@interface ImagePickerManager : NSObject <RCTBridgeModule, UINavigationControllerDelegate, UIActionSheetDelegate, UIImagePickerControllerDelegate, PHPickerViewControllerDelegate>
```

Comment:
`UIActionSheetDelegate` conformance was declared but not required anywhere. Remove it completely